build_tools/aws-sdk-code-generator/spec/interfaces/client/async_client_spec.rb

```diff
@@ -48,7 +48,7 @@
 
         sent_event = same_client.send_events[0]
         expect(sent_event.headers[":event-type"].value).to eq('BazResult')
-        expect(sent_event.payload.read).to eq("{\"Details\":[\"baz\"]}")
+        expect(sent_event.payload.read).to eq("{\"Result\":{\"Details\":[\"baz\"]}}")
```

Comment:
This change is the change in behavior for implicit payload member.  This is modeled as:
```
    "BazEvent": {
      "type": "structure",
      "members": {
        "Result": { 
          "shape": "Result"
        }
      },
      "event": true
    },
```
We were previously treating result as the implicit payload (even though it was a structure and not a string/blob...)